src/core/components/accordion/accordionItemContent/accordionItemContent.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from '@testing-library/react';
+import { type ComponentPropsWithRef } from 'react';
+import { AccordionContainer } from '../accordionContainer/accordionContainer';
+import { AccordionItem } from '../accordionItem/accordionItem';
+import { AccordionItemContent } from './accordionItemContent';
+
+describe('<AccordionItemContent /> component', () => {
+    const createTestComponent = (props?: ComponentPropsWithRef<typeof AccordionItemContent>) => {
+        const completeProps = { ...props };
+        return (
+            <AccordionContainer type="multiple" defaultValue={['value-key']}>
+                <AccordionItem value="value-key">
+                    <AccordionItemContent {...completeProps} />
+                </AccordionItem>
+            </AccordionContainer>
+        );
+    };
+
+    it('renders without crashing', () => {
```

Comment:
We're actually testing that:
```suggestion
    it('renders the children property', () => {
```